src/styles/main.scss

```diff
@@ -0,0 +1,70 @@
+body {
+  margin: 0;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.calendar {
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
+
+  // .calendar__item
+
+  &__day {
+    box-sizing: border-box;
+    list-style: none;
+    height: 100px;
+    width: 100px;
```

Comment:
The height and width values `100px` are repeated. Consider defining SCSS variables for these values to improve maintainability and readability.